app/templates/index.hbs

```diff
@@ -38,19 +38,19 @@
 <div id='home-crates' class='crate-lists'>
   <section id='new-crates' data-test-new-crates aria-busy="{{dataTask.isRunning}}">
     <h2>New Crates</h2>
-    {{crate-list crates=model.new_crates}}
+    {{crate-list-max crates=model.new_crates}}
   </section>
   <section id='most-downloaded' data-test-most-downloaded aria-busy="{{dataTask.isRunning}}">
     <h2>Most Downloaded</h2>
-    {{crate-list crates=model.most_downloaded}}
+    {{crate-list-name-only crates=model.most_downloaded}}
   </section>
   <section id='just-updated' data-test-just-updated aria-busy="{{dataTask.isRunning}}">
     <h2>Just Updated</h2>
-    {{crate-list crates=model.just_updated}}
+    {{crate-list-newest crates=model.just_updated}}
   </section>
   <section id='most-recently-downloaded' data-test-most-recently-downloaded aria-busy="{{dataTask.isRunning}}">
     <h2>Most Recent Downloads</h2>
-    {{crate-list crates=model.most_recently_downloaded}}
+    {{crate-list-max crates=model.most_recently_downloaded}}
```

Comment:
The "Most Recent Downloads" list is for any version of the crate, like "Most Downloaded" is, so I think this should use `crate-list-name-only` too? What do you think?

Reply:
Ah! Right. I hadn't managed to see this. Thanks!